ts-packages/web/src/components/comment/index.tsx

```
@@ -1,32 +1,49 @@
+import RichTextEditor from './rich-text-editor';
 
 interface CommentProps {
   comment: CommentType;
   onSubmit?: (comment_id: number, content: string) => Promise<void>;
   onLike?: (comment_id: number, value: boolean) => Promise<void>;
 }
 
+const HtmlViewer = ({ html }: { html: string }) => (
+  <div
+    className="prose prose-invert max-w-none"
+    dangerouslySetInnerHTML={{ __html: html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)